### PR TITLE
Dev Tools for easier testing

### PR DESCRIPTION
**Pull request by milafrerichs**
## Description

A dev tools for easier testing of different BE. 
First use-case is for changing the hostname
install it only if not production coming from the awesome [K.C. Dodds](https://kentcdodds.com/blog/make-your-own-dev-tools)
it should be kept out of the bundle when the bundle is built with `NODE_ENV=production` otherwise it will install it but only show on dev or when `dev-tools=true` is set as a query param

## Related Issue
__no issue created_

## How to test it locally
run the app, open the browser. you should see in the bottom left a small settings icon. hover over it, it should open the dev tools pane. select a different hostname. 
if it's not showing add `dev-tools=true` in the query string. 

run the app with `NODE_ENV=production` and it should not show the settings icon.

https://deploy-preview-100--wazimap-staging.netlify.app/?dev-tools=true

## Screenshots
Bildschirmfoto 2020-10-05 um 07 45 12
Bildschirmfoto 2020-10-05 um 07 45 05

### Added
* added `svelte-dev-tools` component 
### Updated
* changed hostname to use `process.env.HOSTNAME` or query string

## Checklist

- [x]  🚀 is the code ready to be merged and go live?
- [x]  🛠 does it work (build) locally

### Pull Request

- [x]  📰 good title
- [x]  📝good description
- [ ]  🔖 issue linked
- [x]  📖 changelog filled out

### Commits

- [x]  commits are clean
- [x]  commit messages are clean

### Code Quality

- [x]  🚧 no commented out code
- [x]  🖨 no unnecessary logging
- [x]  🎱 no magic numbers
- [ ]  ⚙️ ran jslint
- [ ]  🧰 ran codeclimate locally

### Testing

- [ ]  ✅ added (appropriate) unit tests
- [ ]  💢 edge cases in tests were considered
- [ ]  ✅ ran tests locally & are passing

### CODE REVIEW

**Comment by adieyal** [comment]
That is really cool

File: src/css/webflow.css [hunk 1]
```diff
@@ -1807,3 +1807,6 @@ textarea.w-select {
 .w-condition-invisible {
   display: none !important;
 }
+#dev-tools {
```
Comment (jbothma) [inline src/css/webflow.css:1810]: This will need to be added on the webflow side, right?

Comment (milafrerichs) [reply src/css/webflow.css:1810]: yes indeed. I could add it to the svelte component but I like it to be not opinionated how you would use it.
